.golangci.yaml

```diff
@@ -0,0 +1,30 @@
+run:
+  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  deadline: 90s
+
+linters-settings:
+  gocyclo:
+    # minimal code complexity to report, 30 by default (but we recommend 10-20)
+    min-complexity: 25
+  depguard:
+    list-type: blacklist
+    packages:
+      - golang.org/x/net/context
+      - github.com/gogo/protobuf/proto
+      - github.com/google/certificate-transparency-go/trillian
+
+linters:
+  disable-all: true
+  enable:
+    - gocyclo
+    - gofmt
+    - goimports
+    - golint
+    - megacheck
+    - misspell
+    - govet
+    - depguard
+    - deadcode
+    - ineffassign
+    - varcheck
+    # - license
```

Comment:
nit: having something commented out just leaves a puzzle for future maintainers; could you convert to a full comment (e.g. "License check is local and so gets run by hand in scripts/presubmit.sh")